Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.py

```diff
@@ -106,7 +113,7 @@ def incidents_list(self, timeout: int, limit: int = MAX_ENTRIES, status: Optiona
     @logger
     def update_incident(self, incident_id: int, status: Optional[str], assigned_to: Optional[str],
                         classification: Optional[str],
-                        determination: Optional[str], tags: Optional[List[str]], timeout: int) -> Dict:
+                        determination: Optional[str], tags: Optional[List[str]], timeout: int, comment: str) -> Dict:
         """
         PATCH request to update single incident.
         Args:
```

Comment:
Pleae add the comment argument description.

Reply:
Done in the latest commit